### PR TITLE
Fabo/fix toolbar

### PR DESCRIPTION
**Pull request by faboweb**
Closes #ISSUE

**Description:**

Toolbar was broken and on the left

Thank you! 🚀

---

For contributor:

- [ ] Added entries in `CHANGELOG.md` with issue # and GitHub username
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by fedekunze** [comment]
screenshots ? 😃

**Comment by jbibla** [comment]
i think there are other `refreshables` elsewhere

**Comment by faboweb** [comment]
I brought back refresh. We can remove it once we trust our software that it updates correctly in all the cases. Refresh should not stay in in the future.

File: app/src/renderer/components/common/ToolBar.vue [hunk 1]
```diff
@@ -45,6 +45,19 @@ import TmBtn from "common/TmBtn"
 export default {
   name: `tool-bar`,
   components: { TmBtn },
+  props: {
+    refresh: {
```
Comment (jbibla) [inline app/src/renderer/components/common/ToolBar.vue]: i think this is broken @faboweb

Comment (jbibla) [reply app/src/renderer/components/common/ToolBar.vue]: nested object in props

**Pull request merged**